src/docker/engine.rs

```diff
@@ -108,6 +108,11 @@ impl Engine {
             .map(|s| bool_from_envvar(&s))
             .unwrap_or_default()
     }
+
+    #[must_use]
+    pub fn has_buildkit() -> bool {
+        !bool_from_envvar("CROSS_CONTAINER_ENGINE_NO_BUILDKIT")
```

Comment:
aha... the classic

this functions takes the value of the envvar, not the key...

Reply:
we should look into making this harder to missuse

Reply:
I thought for consistency we should keep it as `0`, `1`, `true`, `false`, etc.

Reply:
happened before here: https://github.com/cross-rs/cross/pull/997

Reply:
Ahhhhh. That is confusing, and I was the one who added that function.

Reply:
I've fixed the invocation and also added documentation for `bool_from_envvar`:

```rust
// this takes the value of the environment variable,
// so you should call `bool_from_envvar(env::var("FOO"))`
pub fn bool_from_envvar(envvar: &str) -> bool {
    if let Ok(value) = bool::from_str(envvar) {
        value
    } else if let Ok(value) = i32::from_str(envvar) {
        value != 0
    } else {
        !envvar.is_empty()
    }
}
```

```rust
    #[must_use]
    pub fn has_buildkit() -> bool {
        !env::var("CROSS_CONTAINER_ENGINE_NO_BUILDKIT")
            .map(|x| bool_from_envvar(&x))
            .unwrap_or_default()
    }
```